lib/mayaUsd/ufe/Global.cpp

```diff
@@ -20,33 +20,31 @@
 #include <mayaUsd/ufe/ProxyShapeHandler.h>
 #include <mayaUsd/ufe/ProxyShapeHierarchyHandler.h>
 #include <mayaUsd/ufe/StagesSubject.h>
+#include <mayaUsd/ufe/UfeVersionCompat.h>
```

Comment:
Need this file to use the _UFE_V2_ macro. The changes in header files below came from a cleanup and then re-run of clang-format.